### ISSUE TITLE
Need ability to set priority

### ISSUE THREAD

**Issue by eriknelson** [konveyor/github-actions#6]
I need the ability to set the priority using the prow commands so that I'm
able to set a priority and sort our prioritized backlog by that field.
We'd like to make github the source of truth, so priority should be set
on github issues and flow to the Jira that it's reconciling against.
To make things easy, let's mirror the Jira priorities as labels in GH:

priority/blocker
priority/critical
priority/major
priority/normal
priority/minor

An issue should only ever be considered to be completely triaged when it has
a priority set.

**Comment by eriknelson** [comment]
Wonder if we can map the native prow priority levels to jira priorities, or if it just accepts an arbitrary string as a priority?

**Comment by djzager** [comment]
Dropping myself from assignees since I'm not actively working on it.

**Comment by eriknelson** [comment]
@djzager yeah I'd like to map a gh priority label to the actual priority field in Jira because it allows for sorting by that field and jira has an understanding of its ordinality, which you can't do with a label

**Comment by eriknelson** [comment]
Assigning myself to take this one
